WorkflowCombine/Testing/WorkerTesting.swift

```diff
@@ -62,9 +63,9 @@ extension RenderTester {
                 guard !workflow.worker.isEquivalent(to: expectedWorker) else {
                     return
                 }
-                XCTFail(
+                reportIssue(
```

Comment:
There's no Apple-provided way to do this?

Reply:
by 'this', what do you mean? record an issue in either testing framework?